spec/restaurant_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "restaurant"
+
+describe Restaurant do
+
+  it "creates a default menu when initialized" do
+    expect(subject.menu).to eq({ "starter" => 3, "main" => 6, "drinks" => 2 })
+  end
+
+  describe ".show_menu" do
+    it "responds to show_menu" do
+      expect(subject).to respond_to :show_menu
+    end
+
+    it "shows the default menu when initalized" do
+      expect(subject.show_menu).to eq({ "starter" => 3, "main" => 6, "drinks" => 2 })
```

Comment:
Metrics/LineLength: Line is too long. [85/80]<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.